### PR TITLE
BOM-2027 : Fix Python 3.5 test failures

### PR DESCRIPTION
**Pull request by mzulqarnain1**
Relevant JIRA : https://openedx.atlassian.net/browse/BOM-2027

### CODE REVIEW

**Comment by mzulqarnain1** [comment]
@morenol Please review.

**Review by morenol — APPROVED** [review]
LGTM,

Could you add to the deploy section of travis a parameter that only upload this to PyPI if TARGETS="quality"?

**Comment by mzulqarnain1** [comment]
> LGTM,
> 
> Could you add to the deploy section of travis a parameter that only upload this to PyPI if TARGETS="quality"?

@morenol Done.

File: requirements/base.txt [hunk 1]
```diff
@@ -9,22 +9,24 @@ billiard==3.6.3.0         # via celery
 celery==4.4.7             # via -c requirements/constraints.txt, -r requirements/base.in
 certifi==2020.6.20        # via requests
 chardet==3.0.4            # via requests
-configparser==4.0.2       # via importlib-metadata
-contextlib2==0.6.0.post1  # via importlib-metadata, zipp
-edx-rest-api-client==1.5.0  # via -c requirements/constraints.txt, -r requirements/base.in
+django-waffle==2.0.0      # via edx-django-utils
+django==2.2.16            # via edx-django-utils
```
Comment (awais786) [inline requirements/base.txt]: Please add a constraint here django<2.0 and run upgrade.

**Review by morenol — APPROVED** [review]

**Review by awais786 — APPROVED** [review]

**Pull request merged**